### PR TITLE
[KOGITO-5578] Changing listener approach

### PR DESCRIPTION
**Pull request by fjtirado**
QuarkusMultiCloudEventReceiver is not longer needed once we are
generating the listeners, by just adding annotation

### CODE REVIEW

**Review by danielezonca — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: kogito-codegen-modules/kogito-codegen-core/src/main/resources/class-templates/config/GlobalObjectMapperQuarkusTemplate.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 
+import io.cloudevents.jackson.JsonFormat;
```
Comment (ricardozanini) [inline kogito-codegen-modules/kogito-codegen-core/src/main/resources/class-templates/config/GlobalObjectMapperQuarkusTemplate.java:28]: This dependency only comes with the `kogito-addons-cloudevents`. Won't be a problem? How about injecting the `GlobalObjectMapper` into the Quarkus CloudEvents addon and register the module there?

**Review by ricardozanini — COMMENTED** [review]
I think we might have a problem with CE module. It's not on the classpath without the addon. :thinking: Or it should not..

**Review by fjtirado — COMMENTED** [review]
Opened https://issues.redhat.com/browse/KOGITO-5590 to deal with the point raised by @ricardozanini

File: kogito-codegen-modules/kogito-codegen-core/src/main/resources/class-templates/config/GlobalObjectMapperQuarkusTemplate.java (hunk 1 above)
Comment (fjtirado) [reply kogito-codegen-modules/kogito-codegen-core/src/main/resources/class-templates/config/GlobalObjectMapperQuarkusTemplate.java]: Opened https://issues.redhat.com/browse/KOGITO-5590 to deal with that

**Comment by fjtirado** [comment]
jenkins retest app examples

**Comment by fjtirado** [comment]
jenkins run app examples

**Comment by fjtirado** [comment]
jenkins rerun examples tests

**Comment by fjtirado** [comment]
jenkins rerun examples tests

**Pull request merged**